frontend/src/Interface/FarmDetailsInterface.tsx

```diff
@@ -6,7 +6,7 @@
  * @author @GDamaso
  */
 interface FarmDetailsInterface {
-  Year: string;
+  Year: number;
```

Comment:
Hey bud, sorry I didn't clear this out in the docs, but we should follow the current .nmp file type implementation so it's easier to convert from/to json. The way nmp stores Year is as a string, that's why I chose it instead of a number. Here's an example of a .nmp file:

```json
{
  "farmDetails": {
    "Year": "2024",
    "FarmName": "FarmName",
    "FarmRegion": 21,
    "FarmSubRegion": null,
    "SoilTests": null,
    "TestingMethod": "11",
    "Manure": null,
    "HasSelectedFarmType": true,
    "ImportsManureCompost": true,
    "HasAnimals": false,
    "HasDairyCows": false,
    "HasBeefCows": false,
    "HasPoultry": false,
    "HasMixedLiveStock": false,
    "HasHorticulturalCrops": true,
    "HasBerries": true,
    "LeafTests": null,
    "LeafTestingMethod": "0",
    "UserJourney": 7
  },
 ```